AutoFocusV2 Fixed IPv6 recognized as IPv4 (#15381)
demisto/content: Packs/AutoFocus/ReleaseNotes/1_3_11.md

#### Integrations
##### Palo Alto Networks AutoFocus v2

> - Fixed an issue where indicators *IPv6* was recognized as *IPv4* in Human Readable.

Reviewer: ```suggestion
- Fixed an issue where indicators of type *IPv6* were shown as *IPv4* in the result.
```
